tests/dedicatedvstests/l7_dedicated_crd_test.go

```diff
@@ -485,7 +485,7 @@ func TestFQDNsCountForAviInfraSettingWithDedicatedShardSize(t *testing.T) {
 	g.Eventually(func() bool {
 		_, found := mcache.VsCacheMeta.AviCacheGet(vsKey)
 		return found
-	}, 50*time.Second).Should(gomega.Equal(false))
+	}, 50*time.Second, 5*time.Second).Should(gomega.Equal(false))
```

Comment:
why are increasing the polling period from 10 ms to 5 seconds?

Reply:
to decrease the load on cpu